sql/core/src/test/scala/org/apache/spark/sql/execution/SQLViewSuite.scala

```diff
@@ -167,6 +167,12 @@ abstract class SQLViewSuite extends QueryTest with SQLTestUtils {
       assertAnalysisError(
         s"ALTER TABLE $viewName DROP PARTITION (a='4', b='8')",
         s"$viewName is a temp view. 'ALTER TABLE ... DROP PARTITION ...' expects a table")
+      assertAnalysisError(
+        s"ALTER TABLE $viewName SET TBLPROPERTIES ('p' = 'an')",
+        s"$viewName is a temp view. 'ALTER TABLE ... SET TBLPROPERTIES' expects a table")
+      assertAnalysisError(
```

Comment:
cc @MaxGekk we should add tests here.

Reply:
And I guess to `HiveDDLSuite` too, right? I just wonder if we test v2 and v1 Hive external catalog, should we test v1 In-Memory catalog in that case?

Reply:
I don't think so. It's UT and we know how the code works. The catalog implementation doesn't matter. Testing it once with whatever catalog is good enough.